### PR TITLE
alt for img tags

### PR DESCRIPTION
**Pull request by Codooze**
# Milestone VI
- Worked on accessibility, covering the next points:

- [x] Page titles  - no issues found
- [x] Image text alternatives** (alt text)
- [x] Color contrast  - no issues found
- [x] Text headings  - no issues found
- [x] Resize  - no issues found
- [x] Moving content  - no issues found
- [x] Multimedia  - no issues found
- [x] The basic structure of the page  - no issues found

### CODE REVIEW

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Mehr-o-maah  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that does not have your partners commit to the codebase as its a pair programming activity

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -83,7 +86,10 @@ <h3>Tonic</h3>
       </div>
       <div class="cards">
         <div class="card_img_container">
-          <img src="assets/cardImgs/Snapshoot Portfolio-1.png" alt="" />
+          <img
+            src="assets/cardImgs/Snapshoot Portfolio-1.png"
+            alt="Project preview image"
+          />
```
Comment (Mehr-o-maah) [inline index.html:92]: please use pair programming for this milestone

File: index.html [hunk 2]
```diff
@@ -58,7 +58,10 @@ <h2 class="h2_gray_title">Glad to see you!</h2>
     <section id="works_container">
       <div class="cards">
         <div class="card_img_container">
-          <img src="assets/cardImgs/Snapshoot Portfolio.png" alt="" />
+          <img
+            src="assets/cardImgs/Snapshoot Portfolio.png"
+            alt="Project preview image"
```
Comment (Mehr-o-maah) [inline index.html:63]: pair

**Review by Codooze — COMMENTED** [review]
testing

**Pull request merged**